sample.json

```diff
@@ -1,934 +1,938 @@
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x6869757b",
-          "module_index":1,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x683a1329",
-          "module_index":1,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x67751830",
-          "module_index":1,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x67750523",
-          "module_index":1,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x6774ff38",
-          "module_index":1,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x6774fe77",
-          "module_index":1,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x6774b300",
-          "module_index":1,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x6774c7ac",
-          "module_index":1,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x6836ed3f",
-          "module_index":1,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x67bd5ba8",
-          "module_index":1,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x67bd5b68",
-          "module_index":1,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x67bacd6e",
-          "module_index":1,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x67bacafe",
-          "module_index":1,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x6983403a",
-          "module_index":1,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x6836ecfd",
-          "module_index":1,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x67bd5ba8",
-          "module_index":1,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x67bd5b68",
-          "module_index":1,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x69833e8b",
-          "module_index":1,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x69836eb3",
-          "module_index":1,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x1208530",
-          "module_index":0,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x12066ff",
-          "module_index":0,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x12050da",
-          "module_index":0,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x75691173",
-          "module_index":63,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x76e9b3f4",
-          "module_index":75,
-          "trust":"frame_pointer"
-        },
-        {
-          "ip":"0x76e9b3c7",
-          "module_index":75,
-          "trust":"frame_pointer"
+  "payload": {
+    "stack_traces": {
```

Comment:
I added to top-most levels ("payload" and "stack_traces") so this looked like a crash ping. That meant I had to reindent everything in the file. Otherwise there aren't any changes.